frontend/js/helpers/Account.js

```diff
@@ -57,28 +57,33 @@ function updateUI(account,balance=0) {
   
   
     //when addAcc clicked
-    $("#addAcc").submit(async (event) => {
-        event.preventDefault();
-        const newName = $("#accountInput").val().trim();
-        const result = await Post("accounts", {
-            newAccount: newName,
-        });
-  
-        //show result in UI and handle server error
-        if (!showNotification(result)) {
-            return;
-        }
-  
-        // Update the UI
-        updateUI({
-            id: result.id,
-            username: newName,
-        });
-  
-        // Clear input
-        $("#accountInput").val("");
-        $("#accBtn").attr("disabled", true);
-    });
+   // Event listener for when the "addAcc" form is submitted
+$("#addAcc").submit(async (event) => {
+  event.preventDefault();
+  const newName = $("#accountInput").val().trim();
+
+  const accounts = await Get("accounts");
+
+  if (accounts.some(account => account.username === newName)) {
```

Comment:
Sorry for late reply but would you move these validation into change event method?
To make consistency with category validation, would you change alert notification to showing message like category one?